module/zfs/zvol.c

```diff
@@ -2010,7 +2010,7 @@ zvol_remove_minors_impl(const char *name)
 static void
 zvol_remove_minor_impl(const char *name)
 {
-	zvol_state_t *zv, *zv_next;
+	zvol_state_t *zv = NULL, *zv_next;
```

Comment:
`zv` is always initialized at 2030 with the value of `list_head(&zvol_state_list)` and yes, `zv_next` might not be initialized if the list is empty, in which case `zv_next` is not used.